scan-apk/action.yaml

```diff
@@ -70,7 +70,7 @@ runs:
         tag: localhost:1234/apk-scan
 
     - id: grype-scan
-      uses: anchore/scan-action@ecfd0e98932e57ea8f68f29c4f418fc41a8194db
+      uses: anchore/scan-action@24fd7c9060f3c96848dd1929fac8d796fb5ae4b4
```

Comment:
can you add the version tag that this git hash is pointing to? will make easier for the next time

Reply:
Sure, added